src/my_project/tagging/metadata_controller.py

```
@@ -428,18 +436,18 @@ def request_disambiguation(
-    downloads_dir = ROOT_DIR / "0 - Downloads"
+def run_tagger(display: QMainWindow, config_manager: ConfigManager):
+    downloads_dir = config_manager.config.comicsroot.path / "0 - Downloads"
     for path in downloads_dir.rglob("*"):
         if path.is_dir() and path.name in EXCLUDE:
             continue
         if path.is_file() and any(
             path.name.lower().endswith(ext) for ext in VALID_EXTENSIONS
         ):
-            logging.info(f"Starting to process {path.name}")
-            with RepoWorker() as worker:
+            logger.info(f"Starting to process {path.name}")
```

Ruff (0.16.1)

[warning] Logging statement uses f-string

(G004)